From a59e624753e1a7864f86c285c25a4f519bca376f Mon Sep 17 00:00:00 2001
From: Sergio Prado <sergio.prado@toradex.com>
Date: Thu, 28 Sep 2023 10:32:46 +0000
Subject: [PATCH 1/2] Add TorizonCore specific groups

Upstream-Status: Inappropriate [configuration]

Signed-off-by: Rogerio Guerra Borin <rogerio.borin@toradex.com>
Signed-off-by: Eduardo Ferreira <eduardo.barbosa@toradex.com>
Signed-off-by: Sergio Prado <sergio.prado@toradex.com>
Signed-off-by: Stefan Agner <stefan.agner@toradex.com>
---
 group.master | 6 ++++++
 1 file changed, 6 insertions(+)

diff --git a/group.master b/group.master
index e54fd1d..0fa81a3 100644
--- a/group.master
+++ b/group.master
@@ -35,9 +35,15 @@ sasl:*:45:
 plugdev:*:46:
 kvm:*:47:
 sgx:*:48:
+gpio:*:49:
 staff:*:50:
+i2cdev:*:51:
+spidev:*:52:
+pwm:*:54:
 games:*:60:
 shutdown:*:70:
 wheel:*:80:
+nobody:*:99:
 users:*:100:
+render:*:103:
 nogroup:*:65534:
-- 
2.25.1

